src/main/java/com/softserve/demo/model/Customer.java

```diff
@@ -45,6 +50,16 @@
     private List<Offer> offers;
 
     @JsonBackReference(value = "order_customer")
-    @OneToMany(fetch = FetchType.LAZY, mappedBy = "customer",cascade = CascadeType.REMOVE)
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "customer", cascade = CascadeType.REMOVE)
     private List<Order> orders;
+
+    @ManyToMany(cascade = {
```

Comment:
Why PERSIST and MERGE? may be ALL? Guess we need to remove favourites when customer is removed